### PR TITLE
Update github URLs

### PR DESCRIPTION
**Pull request by keithc-ca**
Change all occurrences of https://github.com/AdoptOpenJDK/openjdk-tests to https://github.com/adoptium/aqa-tests.

### CODE REVIEW

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -41,7 +41,7 @@ def setupEnv() {
 
 	env.SPEC = "${SPEC}"
 
-	ADOPTOPENJDK_REPO = params.ADOPTOPENJDK_REPO ? params.ADOPTOPENJDK_REPO : "https://github.com/AdoptOpenJDK/openjdk-tests.git"
+	ADOPTOPENJDK_REPO = params.ADOPTOPENJDK_REPO ? params.ADOPTOPENJDK_REPO : "https://github.com/adoptium/aqa-tests.git"
```
Comment (gdams) [inline buildenv/jenkins/JenkinsfileBase:44]: Can we change `ADOPTOPENJDK_REPO` to `ADOPTIUM_REPO`?

Comment (smlambert) [reply buildenv/jenkins/JenkinsfileBase:44]: Not required (and then would want to update other variable names... and regenerate all test jobs across all Jenkins servers...).

Comment (keithc-ca) [reply buildenv/jenkins/JenkinsfileBase:44]: Ok, I'll leave the variable names alone for now.

**Comment by smlambert** [comment]
@llxia - we can coordinate a full regeneration of test pipelines today across "all" servers (adopt/openj9/internal), once this is merged.  

Do you want to update the variable names as part of this PR? (ADOPTOPENJDK_REPO/ADOPTOPENJDK_BRANCH ?), I do not really care if we rename them or not, so will defer to what you want for it.  I think we will regen once this and https://github.com/adoptium/aqa-tests/pull/2612 are merged.  

@keithc-ca - It will be useful to look for any other references to openjdk-tests in the repo, to ensure the repo rename is handled properly everywhere.

**Comment by llxia** [comment]
It looks like it is safe to merge this PR without regenerating all test jobs. Grinder links:
https://ci.adoptopenjdk.net/job/Grinder/741/
https://ci.adoptopenjdk.net/job/Grinder/742/
https://ci.adoptopenjdk.net/job/Grinder/743/

image

This reason is that this PR does not change root folder name (openjdk-tests-> aqa-tests), such as:
https://github.com/adoptium/aqa-tests/blob/2bc066b7ca739976c045aa3f0dc6cea44f0a6df2/buildenv/jenkins/testJobTemplate#L139
https://github.com/adoptium/aqa-tests/blob/2bc066b7ca739976c045aa3f0dc6cea44f0a6df2/buildenv/jenkins/testJobTemplate#L394
https://github.com/adoptium/aqa-tests/blob/f5ca2f240e7ab1155a669c9d64aa8840d701dba5/buildenv/jenkins/JenkinsfileBase#L16

In summary, I think we can get this PR merged and update the folder in a future PR.

**Review by smlambert — APPROVED** [review]

**Review by llxia — APPROVED** [review]
LGTM
